app/views/home/index.html.erb

```diff
@@ -1 +1,6 @@
-<%= render partial: "signup_form" %>
```

Comment:
removed the partial to be consistent with new home page layout recieved today from designer